client/state/data-layer/wpcom/sites/posts/likes/mine/delete/index.js

```diff
@@ -23,8 +23,13 @@ export function fromApi( response ) {
 	};
 }
 
-export const fetch = action =>
-	http(
+export const fetch = action => {
+	const query = {};
```

Comment:
is this used?

Reply:
I think when you commented, it wasn't in the diff yet.
Now used in line 38